src/botPage/view/blockly/blocks/tools/total_runs.js

```diff
@@ -9,5 +10,14 @@ Blockly.Blocks.total_runs = {
         this.setTooltip(translate('Returns the number of runs since the beginning'));
         this.setHelpUrl('https://github.com/binary-com/binary-bot/wiki');
     },
+    onchange: function onchange(ev) {
```

Comment:
You can change this to the suggestion below, there is no need to name the function twice.

```suggestion
    onchange: function(ev) {
```